lib/interface.ts

```diff
@@ -3,16 +3,22 @@
 import restify = require('restify');
 import bunyan = require('bunyan');
 import BAPI = require('./blpapi-wrapper');
+import Session = require('./apisession/session');
 
 export interface IOurRequest extends restify.Request {
     clientCert?: any;
     blpSession: BAPI.Session;
+    apiSession?: Session;
+}
+
+export interface IBufferedData<T> {
```

Comment:
I see javascript interface is a contract across files, so the impression is any user code who expect `IBufferedData` will import the interface from `interface.ts`, but the data won't necessary come from `historical-buffer-manager`.